scripts/release/issue-list.sh

```diff
@@ -40,9 +40,22 @@ function milestone_issues()
   # Remove enclosing quotes on each line
   issue_list=`echo "$issue_list" | tr -d \"`
 
+  # Replace \ which is left over from above command with "(double quote) and suppress warning
+  issue_list=`echo "$issue_list" | tr '\' '"' 2> /dev/null`
+
   # Adjust the issue links
   issue_list=`echo "$issue_list" | sed -e s/api.github.com.repos/github.com/g`
-  echo "$issue_list"
+
+  features="# Features"
```

Comment:
> Looking good to me. I think we should add some empty lies though as @LalatenduMohanty suggested. Basically empty lines around the headers. Other than that ok.

It is not required as on release page, it is visible as required. See here https://github.com/minishift-bot/minishift/releases/tag/v1.2.10